server/views/pages/add-steps.njk

```diff
@@ -77,9 +77,9 @@
     </div>
 </div>
 
-<form id="create-goal-form" method="POST">
+<form id="add-steps-form" method="POST">
     <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
-    <input type="hidden" id="_areaOfNeed" value="{{ data.areaOfNeed }}"/>
```

Comment:
Did this have any use or was it just leftover?

Reply:
the form had been copied/pasted from create goal - it's why the form id was wrong too

Reply:
so no use, leftover from the copy/paste